### PR TITLE
Add breeze variants

### PR DESCRIPTION
**Pull request by dfryer1193**
## Description
Adds variations for Southern (left-handed) and Summer (dual-cluster) breeze keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/afternoonlabs/breeze/keymaps/southpaw/keymap.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 eithanshavit
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/breeze/keymaps/southpaw/keymap.c]: Can this be included within the default keymap instead? So the default keymap for the southern breeze layout is this?

File: keyboards/afternoonlabs/breeze/keymaps/summer/keymap.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 eithanshavit
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/breeze/keymaps/summer/keymap.c]: Similar question, maybe we can consider adding this to the default keymap instead.

File: keyboards/afternoonlabs/breeze/rev0/info.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "width": 19,
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/breeze/rev0/info.json]: Is it necessary to update the rev0?

**Review by eithanshavit — COMMENTED** [review]
This is really cool @dfryer1193! Some thoughts about the new keymaps

File: keyboards/afternoonlabs/breeze/rev0/info.json (hunk 3 above)
Comment (dfryer1193) [reply keyboards/afternoonlabs/breeze/rev0/info.json]: Eh, the build breaks if we don't, and it's relatively easy to do, so /shrug - We don't have to if you don't want to.

File: keyboards/afternoonlabs/summer_breeze/rev1/info.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
+{
+    "keyboard_name": "Summer Breeze",
+    "url": "afternoonlabs.com/breeze",
+    "productId": "0x0003",
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/summer_breeze/rev1/info.json]: This should be 0x0004

File: keyboards/afternoonlabs/summer_breeze/rev1/config.h [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x616C
+#define PRODUCT_ID 0x0003
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/summer_breeze/rev1/config.h]: This should be 0x0004

File: keyboards/afternoonlabs/southern_breeze/rev1/info.json [hunk 6]
```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Southern Breeze",
+    "url": "afternoonlabs.com/breeze",
+    "productId": "0x0002",
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/southern_breeze/rev1/info.json]: This should be 0x0005

File: keyboards/afternoonlabs/southern_breeze/rev1/config.h [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x616C
+#define PRODUCT_ID 0x0002
```
Comment (eithanshavit) [inline keyboards/afternoonlabs/southern_breeze/rev1/config.h]: This should be 0x0005

**Pull request closed without merging**